hub-templates/daskhub/values.yaml

```diff
@@ -51,6 +50,9 @@ basehub:
       extraEnv:
         # The default worker image matches the singleuser image.
         DASK_GATEWAY__CLUSTER__OPTIONS__IMAGE: '{JUPYTER_IMAGE_SPEC}'
+        DASK_DISTRIBUTED__DASHBOARD_LINK: '/user/{JUPYTERHUB_USER}/proxy/{port}/status'
```

Comment:
Awesome - another thing I'm hoping we can extract and merge independently :)

Reply:
Even without this, I see dask-dashboard is functional, and does not use jupyter-server-proxy 
<img width="701" alt="image" src="https://user-images.githubusercontent.com/30430/130427056-d9d46706-ca0c-4390-a3c0-7916925d9896.png">

Is this necessary for dask-gateway? The scheduler (and hence the dashboard process) lives in another pod anyway.

Reply:
Im generally unsure about this config, it is a migrated pattern so probably no longer relevant like this.